en/middleware/micrortps.md

```diff
@@ -0,0 +1,338 @@
+# PX4-FastRTPS bridge
+
+
+This bridge adds communication capabilities between a **PX4 Autopilot** and a **Fast RTPS** application through serial ports or
+UDP sockets using **CDR serialization**. The goal is to provide a DDS (Data Distribution Service) interface to PX4. This interface allows sharing information also with the upcoming release of ROS (Robot Operating System), ROS2.
+
+DDS is an standard from the OMG (Object Management Group) providing a real-time pub/sub middleware widely used in aerospace, defense and IoT applications, while in Robotics has been adopted as the middleware for ROS2.
+
+![alt text](res/1_general-white.png)
+
+## Automatic code generation
+
+The support for the functionality is mainly done within three new (automatically generated) code blocks.
+
+-  CDR serialization support is generated for every uORB topic adding a interfaces what for *sensor_combined.msg* topic looks like this:
```

Comment:
I THINK you mean 
"The toolchain enables CDR serialisation support by adding two interface methods for every uORB topic: `serialize_*` and  `deserialize_*`. For the *sensor_combined.msg* topic the methods look like this:"